### DISCUSSION TITLE
Using baresip without sip

### DISCUSSION THREAD

**Discussion by gbernaldo82** [baresip/baresip#3065] [Q&A] [answered]
Hi, good morning!

I have a question regarding using barseip without sip. I mean, I have one application which handles SIP sessions and that application sends me notifications about RTP Flows (just notifications about starting/stopping a new RTP Flows, ...) but that application does not handle RTP Audio. I have to delegate that task to another task (that's why I thought baresip could help me with this problem) The audio comes in RTP format from another place (with their source address, and source port, payload type, ssrc, etc...) 
In the other hand I have to send audio in RTP format when the application sends me a notification. The source of the audio comes from my microphone when I get a notification. 

My question is if there is any possibility to use baresip (without SIP) for this kind of scenario. 

Thank you so much in advance!

**Comment by cspiel1** [comment, marked as answer]
Have a look on module multicast in baresip-apps.
- https://github.com/baresip/baresip-apps/blob/main/modules/multicast/multicast.c
- https://github.com/baresip/baresip-apps/blob/main/docs/examples/config
